app.js

```diff
@@ -50,70 +46,129 @@ function randomPick(){
 };
 
 //Stored random numbers
-var itemLeft = randomPick();
-var itemCenter = randomPick();
-var itemRight = randomPick();
+var itemLeft = 0;
+var itemCenter = 0;
+var itemRight = 0;
 console.log(itemLeft);
 console.log(itemCenter);
 console.log(itemRight);
 
-var leftItemPicture = items[itemLeft];
-var centerItemPicture = items[itemCenter];
-var rightItemPicture = items[itemRight];
-console.log(leftItemPicture);
-console.log(centerItemPicture);
-console.log(rightItemPicture);
-//Aaray for random three picks
-var randomThreePicks = [];
-
-//Pick three random numbers to push to aaray
-function itemPick(){
-  for(var i = 0; i < 3; i++){
-    var currentItem = Math.floor(Math.random() * items.length);
-    randomThreePicks.push(currentItem);
+function drawThree(){
+  itemLeft = randomPick();
+  itemCenter = randomPick();
+  itemRight = randomPick();
+
+  //Aaray for random three picks
+  var randomThreePicks = [];
+  randomThreePicks.push(itemLeft);
+  randomThreePicks.push(itemCenter);
+  randomThreePicks.push(itemRight);
+  console.log(randomThreePicks);
+
+  //prevents repeating items per turn
+  while (randomThreePicks.includes(itemLeft)){
+    itemLeft = randomPick();
+  };
+  while (itemCenter === itemLeft || randomThreePicks.includes(itemCenter)){
+    itemCenter = randomPick();
+  };
+  while (itemRight === itemLeft || itemRight === itemCenter || randomThreePicks.includes(itemRight)){
+    itemRight = randomPick();
   }
+  //calling functions
+  drawCenterItem(items[itemCenter]);
+  drawLeftItem(items[itemLeft]);
+  drawRightItem(items[itemRight]);
 };
-itemPick();
-console.log(randomThreePicks);
 
 //Creates image node
```

Comment:
As a challenge, is there a way you can turn these three functions into one? (You do not have to, only if you have time)